adhocracy-plus/assets/scss/_fonts.scss

```diff
@@ -1,41 +1,27 @@
 @font-face {
-    font-family: "SourceSerifPro";
-    src: url("../fonts/SourceSerifPro-Regular.otf");
+    font-family: "Cabin";
+    src: url("../fonts/Cabin-Regular.ttf");
     font-style: normal;
     font-weight: normal;
 }
 
 @font-face {
-    font-family: "SourceSerifPro";
-    src: url("../fonts/SourceSerifPro-It.otf");
+    font-family: "Cabin";
+    src: url("../fonts/Cabin-Italic.ttf");
     font-style: italic;
     font-weight: normal;
 }
 
 @font-face {
-    font-family: "SourceSansPro";
-    src: url("../fonts/SourceSansPro-Regular.otf");
-    font-style: normal;
-    font-weight: normal;
-}
-
-@font-face {
-    font-family: "SourceSansPro";
-    src: url("../fonts/SourceSansPro-It.otf");
-    font-style: italic;
-    font-weight: normal;
-}
-
-@font-face {
-    font-family: "SourceSansPro";
-    src: url("../fonts/SourceSansPro-Semibold.otf");
+    font-family: "Cabin";
+    src: url("../fonts/Cabin-SemiBold.ttf");
     font-style: normal;
     font-weight: 600;
 }
 
 @font-face {
-    font-family: "SourceSansPro";
-    src: url("../fonts/SourceSansPro-Bold.otf");
+    font-family: "Cabin";
+    src: url("../fonts/Cabin-Bold.ttf");
     font-style: bold;
```

Comment:
I assume your remarks regarding `font-style` and `font-weight` are not specific to the new Cabin font but a general issue. As this whole project is forked from a+ we should fix the issues in a+ (https://github.com/liqd/adhocracy-plus/blob/main/adhocracy-plus/assets/scss/_fonts.scss), and then we rebase this repo, so we get the fixes automatically instead of having to fix it in a+ and all forks individually (defakts, diid, kfd, kosmo)